### PR TITLE
Print out the dictionaries

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: osx-learn.sh [hunk 1]
```diff
@@ -46,18 +66,21 @@ esac
...
+# 
+# data=$(cat $database | sort)
+# 
+# echo "$data" > $database
+# 
+# #
+# # Restart AppleSpell.
+# #
+# 
+# killall -e AppleSpell &> /dev/null
```
Comment (wooorm) [inline osx-learn.sh]: Is this part intentional? I think this breaks the writing functionality?

Comment (RichardLitt) [reply osx-learn.sh]: Ha! Sorry, was testing and forgot to remove this. One second.

Comment (wooorm) [reply osx-learn.sh]: :+1:

**Comment by RichardLitt** [comment]
Fixed.

**Comment by wooorm** [comment]
❤️

**Pull request closed without merging**